concent_api/core/templatetags/admin_tags.py

```diff
@@ -26,7 +26,7 @@ def get_active_subtasks_amount():
 
 @register.assignment_tag
 def get_active_subtasks_status():
-    return get_result_uploaded_subtasks().exists()
+    return get_result_uploaded_subtasks().exists()  # pylint: disable=no-member
```

Comment:
Did you try use newest pylint? Becuase some warnings were fixed in pylint and there is no more need to add disables in some cases

Reply:
Yes, but not working

Reply:
Please remember about https://github.com/Code-Poets/coding-style/issues/8

Reply:
@MichalW-CP Please make a list of disabled warnings as stated above.